src/test/cypress/integration/exam/ExamAssessment.spec.ts

```diff
@@ -84,12 +85,12 @@ describe('Exam assessment', () => {
 
         it('Assess a programming exercise submission (MANUAL)', () => {
             cy.login(tutor, '/course-management/' + course.id + '/exams');
-            cy.contains('Assessment Dashboard', { timeout: examDuration }).click();
+            examManagement.openAssessmentDashboard(exam.id, 120000);
```

Comment:
Is there a reason why you changed the timeout from `examDuration` to a static 2 minutes?

Reply:
I just didnt remember this when i refactored all the clicks.